mono/mini/llvm-jit.cpp

```diff
@@ -288,7 +288,7 @@ class MonoLLVMJIT {
 		if (opts == NULL) {
 			// FIXME: find optimal mono specific order of passes
 			// see https://llvm.org/docs/Frontend/PerformanceTips.html#pass-ordering
-			opts = " -simplifycfg -sroa -instcombine -gvn";
+			opts = " -simplifycfg -sroa -early-cse -instcombine -correlated-propagation -instcombine -loop-simplify -lcssa -loop-rotate -licm -instcombine -lcssa -indvars -loop-deletion -instcombine -simplifycfg";
```

Comment:
Nit: simplifycfg is there twice

Reply:
@filipnavara the [last ](https://www.diffchecker.com/flsgO9S8)`-simplifycfg` removes a redundant block generated by previous pass (instcombine). The [first ](https://www.diffchecker.com/Uy5Gs7ah)`-simplifycfg` removes redundant blocks generated by mono 🙂

Reply:
@EgorBo Maybe it's worth adding a comment in the code so nobody removes them in the future?